src/beamlime/config/raw_detectors/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+This config is used during Live Data Reduction detector view.
```

Comment:
Not data-reduction actually, how about something like
```suggestion
This config is used to setup live raw detector views.
```